modular_skyrat/modules/crafting/crafting_skyrat.dm

```diff
@@ -20,7 +20,6 @@
 	result = /obj/item/gun/energy/beam_rifle
 	reqs = list(/obj/item/gun/energy/laser = 1,
 				/obj/item/assembly/signaler/anomaly/flux = 1,
-				/obj/item/assembly/signaler/anomaly/grav = 1,
 				/obj/item/stack/cable_coil = 5,
 				/obj/item/weaponcrafting/gunkit/beam_rifle = 1)
```

Comment:
```suggestion
				/obj/item/weaponcrafting/gunkit/beam_rifle = 1,)
```
putting commas at the end of lists is a good habit to get into for people who want to add things later down the line

Reply:
You should be doing it this way.
```suggestion
				/obj/item/weaponcrafting/gunkit/beam_rifle = 1,
			)
```